app/services/credit_notes/refresh_draft_service.rb

```diff
@@ -45,6 +55,10 @@ def call
 
     private
 
-    attr_accessor :credit_note, :fee
+    attr_accessor :credit_note, :fee, :old_fee_values
+
+    def calculate_item_value(item, old_fee_amount_cents)
+      item.precise_amount_cents.fdiv(old_fee_amount_cents) * fee.amount_cents
```

Comment:
Could you explain this computation? It's not clear what it's doing

Reply:
Added a comment in the code. Basically we didn't change credit note item value with refresh. However it needs to be recalculated based on the ration between old fee value and new fee value